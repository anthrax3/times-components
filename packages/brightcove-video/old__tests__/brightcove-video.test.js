import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

import BrightcoveVideo from "../src/brightcove-video";

const defaultProps = {
  accountId: "[account id]",
  playerId: "[player id]",
  videoId: "[video id]",
  policyKey: "[policy key]",
  poster: { uri: "[poster uri]"},
  paidonly: false,
  width: 300,
  height: 200,
  onVideoPress: () => {}
};

describe("BrightcoveVideo native", () => {

  it("renders correctly", () => {
    const tree = shallow(<BrightcoveVideo {...defaultProps} />);

    expect(tree).toMatchSnapshot();
  })
});
