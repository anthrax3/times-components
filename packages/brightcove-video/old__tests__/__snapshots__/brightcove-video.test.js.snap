// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrightcoveVideo native renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BrightcoveVideo
    accountId="[account id]"
    height={200}
    onVideoPress={[Function]}
    paidonly={false}
    playerId="[player id]"
    policyKey="[policy key]"
    poster={
      Object {
        "uri": "[poster uri]",
      }
    }
    videoId="[video id]"
    width={300}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "accessibilityLabel": "splash-component",
      "activeOpacity": 0.2,
      "children": Array [
        <Image
          source={
            Object {
              "uri": "[poster uri]",
            }
          }
          style={
            Object {
              "height": 200,
              "width": 300,
            }
          }
        />,
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
                "left": 0,
                "position": "absolute",
                "top": 0,
              },
              Object {
                "height": 200,
                "width": 300,
              },
            ]
          }
        >
          <PlayIcon
            icon={null}
          />
        </View>,
      ],
      "onPress": [Function],
      "style": Object {
        "height": 200,
        "width": 300,
      },
      "testID": "splash-component",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "source": Object {
            "uri": "[poster uri]",
          },
          "style": Object {
            "height": 200,
            "width": 300,
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <PlayIcon
            icon={null}
          />,
          "style": Array [
            Object {
              "alignItems": "center",
              "flex": 1,
              "justifyContent": "center",
              "left": 0,
              "position": "absolute",
              "top": 0,
            },
            Object {
              "height": 200,
              "width": 300,
            },
          ],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "icon": null,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "accessibilityLabel": "splash-component",
        "activeOpacity": 0.2,
        "children": Array [
          <Image
            source={
              Object {
                "uri": "[poster uri]",
              }
            }
            style={
              Object {
                "height": 200,
                "width": 300,
              }
            }
          />,
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "flex": 1,
                  "justifyContent": "center",
                  "left": 0,
                  "position": "absolute",
                  "top": 0,
                },
                Object {
                  "height": 200,
                  "width": 300,
                },
              ]
            }
          >
            <PlayIcon
              icon={null}
            />
          </View>,
        ],
        "onPress": [Function],
        "style": Object {
          "height": 200,
          "width": 300,
        },
        "testID": "splash-component",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "source": Object {
              "uri": "[poster uri]",
            },
            "style": Object {
              "height": 200,
              "width": 300,
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": <PlayIcon
              icon={null}
            />,
            "style": Array [
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
                "left": 0,
                "position": "absolute",
                "top": 0,
              },
              Object {
                "height": 200,
                "width": 300,
              },
            ],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "icon": null,
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
